File: jukengkengparent/jukengkeng-sellergoods-interface/src/main/java/com/jukengkeng/sellergoods/service/SellerService.java
package com.jukengkeng.sellergoods.service;

import com.jukengkeng.pojo.TbSeller;
import entity.pageResult;
import entity.result;

import java.util.List;

public interface SellerService {
    //新增商家：商家入驻
    void addSeller(TbSeller tbSeller);
    //分页查询商家
    pageResult queryListSeller(int pageNum, int pageSize, TbSeller tbSeller);
    //根据ID查询商家信息
    TbSeller querySellerByID(String rowid);
    //修改商家审核状态
    void changeStatus(String rowid,String status);
}

File: jukengkengparent/jukengkeng-manager-web/src/main/webapp/js/myAction/service/sellerService.js
app.service("sellerService", function ($http) {
    //查询列表
    this.queryList = function (pageNum, pageSize, searchEntity) {
        return $http.post("../seller/queryListSeller.do?pageNum=" + pageNum + "&pageSize=" + pageSize, searchEntity);
    }

    //根据ID查询商家信息
    this.querySellerByID = function (id) {
        return $http.post("../seller/querySellerByID.do?rowid=" + id);
    }

    //商家审核
    this.changeStatus = function (id,status) {
        return $http.post("../seller/changStatus.do?rowid="+id+"&status="+status);
    }
});